config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-config.yaml

```diff
@@ -296,7 +296,7 @@ presubmits:
   - name: pull-kubernetes-csi-csi-driver-host-path-alpha-1-25-on-kubernetes-1-25
     cluster: eks-prow-build-cluster
     always_run: false
-    optional: true
+    optional: false
```

Comment:
Now that I look at this again, I wonder whether this really makes sense. This is a job which runs alpha features. Should the result of those jobs really be merge-blocking?

Reply:
Thank you for the feedback. I agree that this job, which runs alpha features, shouldn't necessarily be merge-blocking. I'll revert the script to its original state to reflect this.

Reply:
@pohly  I've reverted the chnges. 
Do you think it's reasonable to keep the `TODO` comment? If not, I can remove it. 

```
# https://github.com/kubernetes-csi/csi-driver-host-path/pull/282 has not been merged yet,
# therefore pull jobs which depend on the new deployment flavors have to be optional.
# TODO: remove this check once merged.
```

Reply:
The comment is out-dated, so it would make sense to update the script so that has the right checks and produces the desired output.